app/navigation/SecurityDetailsNavigator.scala

```diff
@@ -29,28 +30,39 @@ class SecurityDetailsNavigator @Inject()() extends Navigator {
   // format: off
   //todo -update when Safety and Security section built
   override protected def normalRoutes: PartialFunction[Page, UserAnswers => Option[Call]] = {
-    case TransportChargesPage(index) => ua => Some(routes.CommercialReferenceNumberController.onPageLoad(ua.id, index, NormalMode))//todo update when Safety and security section built
+    case TransportChargesPage(index) => ua => transportChargesRoute(ua, index, NormalMode)
     case CommercialReferenceNumberPage(index) => ua => Some(routes.AddDangerousGoodsCodeController.onPageLoad(ua.id, index, NormalMode))
     case AddDangerousGoodsCodePage(index) => ua => addDangerousGoodsCodeRoute(ua, index, NormalMode)
-    case DangerousGoodsCodePage(index) => ua => Some(controllers.addItems.routes.ItemsCheckYourAnswersController.onPageLoad(ua.id, index))
+    case DangerousGoodsCodePage(index) => ua => Some(dangerousGoodsCodeRoute(ua, index))
   }
 
   override protected def checkRoutes: PartialFunction[Page, UserAnswers => Option[Call]] = {
     case TransportChargesPage(index) => ua => Some(controllers.addItems.routes.ItemsCheckYourAnswersController.onPageLoad(ua.id, index))
     case CommercialReferenceNumberPage(index) => ua => Some(controllers.addItems.routes.ItemsCheckYourAnswersController.onPageLoad(ua.id, index))
     case AddDangerousGoodsCodePage(index) => ua => addDangerousGoodsCodeRoute(ua, index, CheckMode)
     case DangerousGoodsCodePage(index) => ua => Some(controllers.addItems.routes.ItemsCheckYourAnswersController.onPageLoad(ua.id, index))
-
   }
 
    private def addDangerousGoodsCodeRoute(ua: UserAnswers, index: Index, mode: Mode) =
     (ua.get(AddDangerousGoodsCodePage(index)), ua.get(DangerousGoodsCodePage(index)), mode) match {
       case (Some(true), _, NormalMode) => Some(routes.DangerousGoodsCodeController.onPageLoad(ua.id, index, NormalMode))
-      case (Some(false), _, NormalMode) =>
-        Some(controllers.addItems.routes.ItemsCheckYourAnswersController.onPageLoad(ua.id, index)) //todo -update when Security Trader Details section done
+      case (Some(false), _, NormalMode) => Some(dangerousGoodsCodeRoute(ua, index))
       case (Some(true), None, CheckMode)    => Some(routes.DangerousGoodsCodeController.onPageLoad(ua.id, index, CheckMode))
       case (Some(true), Some(_), CheckMode) => Some(controllers.addItems.routes.ItemsCheckYourAnswersController.onPageLoad(ua.id, index))
       case (Some(false), _, CheckMode)      => Some(controllers.addItems.routes.ItemsCheckYourAnswersController.onPageLoad(ua.id, index))
```

Comment:
```suggestion
      case (Some(_),_, CheckMode) => Some(controllers.addItems.routes.ItemsCheckYourAnswersController.onPageLoad(ua.id, index))
 ```
A minor observation, Can we combine the 2 conditions to one as both point to CYA

Reply:
:+1: